app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError
+        else:
```

Comment:
The use of `else` here is unnecessary. According to the checklist, you can avoid using `else` after `elif` when the previous conditions return or raise an exception. You can directly return the welcome message without the `else` block.